src/main/kotlin/cacophony/controlflow/HardwareRegister.kt

```diff
@@ -60,3 +60,14 @@ val REGISTER_ARGUMENT_ORDER =
         HardwareRegister.R8,
         HardwareRegister.R9,
     )
+
+val RETURN_REGISTER_ORDER =
```

Comment:
i dont know if we want all of there registers for returned value, but this will be easy to change

Reply:
We shall discuss the calling convention further during labs tomorrow. For now this should work.